mintapi/signIn.py

```diff
@@ -138,17 +164,14 @@ def get_email_code(
 
             if diff.seconds > 180:
                 continue
-
+            code = ""
```

Comment:
To my knowledge, there isn't an issue with this section of code at the moment.  I'd like to keep it out of the feature branch to keep scope as limited as possible.